### ISSUE TITLE
Another encoder

### ISSUE THREAD

**Issue by mrlzla** [eldar/pose-tensorflow#40]
Have you tried to train the model with another non-resnet encoder?
For example mobilenet?
It will take realtime on CPU!

**Comment by eldar** [comment]
Hi Vladislav,

I actually did try to train with MobileNets, but accuracy is substantially worse than ResNets (~70% PCK vs 86% of ResNet-50). This is a bit surprising, because the paper reports that accuracy on ImageNet is almost as good as the VGG network. And in my experiments VGG is roughly the same as ResNet-50.

I hope that I am just doing something wrong (different training parameters and so on) and better accuracy can be achieved, but I don't have a lot of time to investigate right now).

Cheers,
Eldar.
